src/server/worker/driver/driver.go

```diff
@@ -550,8 +550,7 @@ func (d *driver) UserCodeEnv(
 	inputs []*common.Input,
 	pachToken string,
 ) []string {
-	result := os.Environ()
-
+	var result []string
```

Comment:
We still need to grab the environment / secrets / etc. that users request in their pipeline spec.  Looks like this doesn't work:

```
                pachctl create pipeline -f - <<EOF                                                                                                                                                                   pipeline:
                    name: pipeline
                  input:
                    pfs:
                      glob: /*
                      repo: input
                  transform:
                    cmd: [ /bin/bash ]
                    stdin:                                                                                                                                                                                               - "env | grep MY_SECRET >/pfs/out/vars"
                    secrets:
                      - name: test-yaml-secret                                                                                                                                                                             env_var: MY_SECRET
                        key: my-key
                EOF                        
```

Reply:
this should be addressed. Please take another look!